client/src/App.jsx

```diff
@@ -8,10 +9,10 @@ class App extends React.Component {
   render () {
     return (
       <div>
-        <Overview />
-        <QandA />
+        {/* <Overview />
+        <QandA /> */}
```

Comment:
for the record while i def understand why commenting these out is useful for working, it's probably best to not commit this change just so it doesn't mess with our content when we go to PR